integration-tests/src/test/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+# TODO replace it with hosts discovery
+hosts = ["50.18.91.185", "54.241.96.75", "54.153.35.114"]
+
+tx-gen {
+    dst = "DAG8y1BQXeVr9n6fh5BZz6KwAs2Ubrjc6VMvzxxw"
+    amount = 1000
+    fee = 10
+    count-total = 100
+    count-per-host = 50
+    key-pair {
```

Comment:
we can assume that all keys will use the same alias/keypass/storepas which we do for all test cases and all testing environments. you just need to provide the path for keys but anyway keys should be taken dynamically from the s3

Reply:
Why taking it from s3 would be better than keeping it next to the config? It's versioned and easier to manage if it's in repo.

Reply:
Will you keep 1k or 10k keys in the confit in the repo?

Reply:
Ok, I see the problem. I will add keys to s3 along with the infra code.

Reply:
Keys are already there